projects/plugins/jetpack/modules/minileven.php

```diff
@@ -1,54 +0,0 @@
-<?php
-/**
- * Deprecated since 8.3.0
- *
- * Originally we created the mobile theme feature as a fall-back
- * when the regular theme did not include a mobile view.
- * Most themes include a mobile view by default now, so the feature is no longer necessary.
- *
- * Visit this page for some alternatives:
- * https://jetpack.com/support/mobile-theme/
- *
- * If you MUST continue to use this module, you can use this standalone plugin as a temporary solution:
- * https://github.com/Automattic/minileven
- * However, we do not recommend it.
- *
- * @deprecated
- * @package automattic/jetpack
- */
-
-/**
- * Deactivate module if it is still active.
- *
- * @since 8.3.0
- */
-if ( Jetpack::is_module_active( 'minileven' ) ) {
-	Jetpack::deactivate_module( 'minileven' );
-}
-
-// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
-_deprecated_file( basename( __FILE__ ), 'jetpack-8.3.0' );
-
-/**
- * Check if we are on mobile.
- */
-function jetpack_check_mobile() {
```

Comment:
This function is used by the following w.org plugins:

- [ ] `gator-cache`: gator-cache.php,676

Reply:
`gator-cache` last updated 3 years ago, 600+ installs. 

`jetpack_check_mobile` [is gated so it will only work](https://plugins.trac.wordpress.org/browser/gator-cache/trunk/gator-cache.php#L671) if `minileven` is in the active module array. I don't immediately recall if we removed that from the active module array when we deprecated minileven. If we did, we can clear this one out.

Reply:
Confirmed that [we will deactivate deprecated modules](https://github.com/Automattic/jetpack/blob/5605eb656d65149dbeebec31077bc13c8ea1d1a2/projects/plugins/jetpack/class.jetpack.php#L2579), so no modern sites should have `minileven` as an active module.